content/doc/book/blueocean/pipeline-editor.adoc

```diff
@@ -18,130 +18,118 @@ endif::[]
 
 = Pipeline Editor
 
-The Blue Ocean Pipeline Editor is the simplest way for anyone to get started with
-creating Pipelines in Jenkins. It's also a great way for existing Jenkins users
-to start adopting Pipeline.
+The Blue Ocean Pipeline Editor is an easy way for anyone to get started with creating Pipelines in Jenkins.
+It's also great for existing Jenkins users to start adopting Pipeline.
 
 include::doc/book/blueocean/_blue-ocean-status.adoc[]
 
-The editor allows users to create and edit Declarative Pipelines, add
-stages, and parallelized tasks that can run at the same time, depending on their
-needs. When finished, the editor saves the Pipeline to a source code repository
-as a `Jenkinsfile`.  If the Pipeline needs to be changed again,
-Blue Ocean makes it easy to jump back into the visual editor to modify the
-Pipeline at any time.
+The editor allows users to create and edit Declarative Pipelines and perform actions such as adding stages or configuring parallel tasks, depending on their needs.
+When the user completes their configuration, the editor saves the Pipeline to a source code repository as a `Jenkinsfile`.
+If the Pipeline requires further modification, Blue Ocean makes it easy to jump back into the visual editor to modify the Pipeline at any time.
 
 video::FhDomw6BaHU[youtube, width=640, height=360, align="center"]
 
-== Starting the editor
+[NOTE]
+.Blue Ocean Pipeline Editor limitations
+====
+The Blue Ocean Pipeline Editor is a great alternative for Jenkins users, but there are some limitations to its functionality:
 
-To use the editor a user must first have
-<<creating-pipelines, created a pipeline in Blue Ocean>>
-or have one or more existing Pipelines already created in Jenkins.
-If editing an existing pipeline, the credentials for that pipeline must allow pushing of
-changes to the target repository.
+* The editor utilizes SCM-based Declarative Pipelines only.
+* User credentials must have write permission.
+* The editor does not have full parity with Declarative Pipeline.
+* The Pipeline is re-ordered and comments are removed.
```

Comment:
I understand that "the comments are removed".  Can you provide more information about the reordering of Pipelines by the editor?  I assume it changes the order of options and environment blocks but does not reorder the stages of execution.
```suggestion
* The Pipeline is re-ordered and comments are removed.
```